Vizzuality/landgriffon
# fix metadata format in materials csv

### General description

Fix metadata format in material csv for a valid ingestion.

### Testing instructions

Go to the data folder and test that the seed-source-data works successfully 

- Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
